### PR TITLE
elk; update with simple config; new orion image

### PR DESCRIPTION
**Pull request by joshuafernandes**
- added elk for logs
- exposing services on native ports now eg (grafana:3000, prom:9090, kibana:5601)
- simplified config and using static node keys 
- volume mounts which will make it a bit easier for people to see whats going on
- use the new orion image

### CODE REVIEW

File: inspect-ibft2.sh [hunk 1]
```diff
@@ -16,80 +16,14 @@ NO_LOCK_REQUIRED=false
 . ./.env
 . ./.common.sh
 
-if [[ ${version} != *"ibft"* ]]; then
```
Comment (EdJoJob) [inline inspect-ibft2.sh]: This check should probably stay in the script

Comment (joshuafernandes) [reply inspect-ibft2.sh]: Not seeing the benefit tbh - its called once only in list.sh which performs the check

**Comment by joshuafernandes** [comment]
> LGTM as it will make the QS easier to use. Just one warning, the Besu version was not set to latest on purpose to make sure Besu updates were not able to break the QS so that's why we sticked to explicit version numbers in the .env file. Now you will have to test the quickstart against the latest version each time Besu latest is updated.

Aye, understand. 'latest' is only for releases at the moment, so should be safeish. I'd like to add this to part of the release checks i.e we spin it up (or some container variation) and check for things - we're using the binaries for most things, but as we move forward we need to check the containers too given the k8s context

File: inspect-ibft2.sh (hunk 1 above)
Comment (EdJoJob) [reply inspect-ibft2.sh]: It was more that it is a "top-level" and therefore "user" script in my mind.  I.e. I could have the quickstart running and want to get the address of the validators again

Comment (EdJoJob) [reply inspect-ibft2.sh]: although the "error" case when running against a non-ibft2 network is fairly clean:
```*************************************
Besu Quickstart latest
*************************************
IBFT 2 Validator Addresses:
----------------------------------

```

Comment (EdJoJob) [reply inspect-ibft2.sh]: but then I think we should also have an `inspect_clique` to get the validators for it as well

File: besu/node_start.sh [hunk 2]
```diff
@@ -11,27 +11,27 @@
...
 BOOTNODE_P2P_PORT="30303"
-
 bootnode_enode_address="enode://${bootnode_pubkey}@${boonode_ip}:${BOOTNODE_P2P_PORT}"
 
-# run with bootnode param
-/opt/besu/bin/besu $@ --bootnodes=$bootnode_enode_address
+p2pip=`awk 'END{print $1}' /etc/hosts`
+
+/opt/besu/bin/besu $@ --bootnodes=$bootnode_enode_address --p2p-host=$p2pip
+
```
Comment (EdJoJob) [inline besu/node_start.sh]: please remove unneeded whitespace

File: .gitignore [hunk 3]
```diff
@@ -7,6 +7,7 @@
 /privacy/besu/data*/private*
 /privacy/besu/data*/uploads
 /privacy/besu/data*/besu.ports
-
+logs/besu/*
+logs/orion/*
```
Comment (EdJoJob) [inline .gitignore]: Can't we replace this with `logs/`?

File: inspect-ibft2.sh (hunk 1 above)
Comment (joshuafernandes) [reply inspect-ibft2.sh]: I've yanked out the inspect.sh file and just put the two lines in the caller - not sure if a similar method exists for clique?

**Pull request merged**